packages/server/src/helpers/calculateMaxMintableForMember.ts

```diff
@@ -0,0 +1,23 @@
+import { DocumentSnapshot } from "@google-cloud/firestore";
+import Big from "big.js";
+
+const RAHA_UBI_WEEKLY_RATE = 10;
```

Comment:
not necessary right now, but maybe worth putting RAHA_UBI_WEEKLY_RATE in shared

Reply:
agreed, I'll leave a TODO for this.